dashboard/app/models/script.rb

```diff
@@ -1246,7 +1246,7 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       assigned_section_id: assigned_section_id,
       hasStandards: has_standards_associations?,
       tts: tts?,
-    }
+    }.with_indifferent_access
```

Comment:
I added `with_indifferent_access` because without it, we were setting both `:title` and `'title'` keys in the hash. the right thing seemed to be happening when the result was serialized into JSON, but I didn't want to rely on that. Added unit tests show that this makes access this hash by string or symbol both work.

Reply:
I'm for adding `with_indifferent_access` here, but how were both the symbol and string getting set into the hash? I thought everything in this method sets it via symbol?

I'm also not sure if we need to unit test that both kinds of access work, unless we have a specific reason that should be part of the contract for this method. (Does existing code already access it in both ways?)

Reply:
> I'm for adding `with_indifferent_access` here, but how were both the symbol and string getting set into the hash? I thought everything in this method sets it via symbol?

it does, but the result of `summarize_i18n` uses string keys and gets merged into the unit group summary here: https://github.com/code-dot-org/code-dot-org/blob/6c43501ed35d3371a13e160bbe34bff4221f152c/dashboard/app/models/unit_group.rb#L298-L300

> I'm also not sure if we need to unit test that both kinds of access work, unless we have a specific reason that should be part of the contract for this method. (Does existing code already access it in both ways?)

good point, I should not need to unit test that `with_indifferent_access` works. I added both keys (symbol and string) to make sure my implementation was working, and then didn't remove them after adding `with_indifferent_access`. I'll remove the string keys now.